engine/runtime-language-epb/src/main/java/org/enso/interpreter/epb/node/JsForeignNode.java

```diff
@@ -34,7 +33,7 @@ Object doExecute(
     if (getArity() - 1 >= 0) System.arraycopy(arguments, 1, positionalArgs, 0, getArity() - 1);
     try {
       return interopLibrary.invokeMember(
-          getForeignFunction(), "apply", arguments[0], new Array(positionalArgs));
+          getForeignFunction(), "apply", arguments[0], new ReadOnlyArray(positionalArgs));
```

Comment:
What does this change imply?

Reply:
Ok, sorry, the `ReadOnlyArray` doc seems to explain it, I didn;t notice it before.

Reply:
Functionally - nothing. We just cannot import Array here since `runtime-language-epb` is a dependency of `runtime` where it is defined.